services/datamanager/builtin/builtin.go

```diff
@@ -686,12 +686,15 @@ func (svc *builtIn) sync() {
 	svc.lock.Unlock()
 
 	stopAfter := time.Now().Add(time.Duration(svc.syncIntervalMins * float64(time.Minute)))
+	if len(toSync) >= 100 {
+		svc.logger.Infof("Number of files to sync: %d", len(toSync))
```

Comment:
Could we change this to something like `starting sync of %d files`. I think it'd be better for the message to more clearly say we are in the process of doing this. We've had customer escalations in the past for non issues because of a log line being slightly misunderstood.

Reply:
yup changed!